### PR TITLE
Refactor user-guide-windows-nodes

### PR DESCRIPTION
**Pull request by yoonian**
- change images of the command output to text.
- split long text in PowerShell command into multiple lines.

For details, refer to https://github.com/kubernetes/website/issues/15833

### CODE REVIEW

File: content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md [hunk 1]
```diff
@@ -143,21 +142,39 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
     kubectl get pods --all-namespaces
     ```
 
-    ![alt_text](../flannel-master-kubeclt-get-pods.png "flannel master kubectl get pods screen capture")
+    The output looks like as follows:
+
+    ```shell
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: nit: This isn't shell output. Leave the format code out and it'll work fine.

File: content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md [hunk 2]
```diff
@@ -143,21 +142,39 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
...
     Verify that the Flannel DaemonSet has the NodeSelector applied.
 
     ```bash
     kubectl get ds -n kube-system
     ```
 
-    ![alt_text](../flannel-master-kubectl-get-ds.png "flannel master kubectl get ds screen capture")
+    The output looks like as follows. The NodeSelector `beta.kubernetes.io/os=linux` is applied.
+
+    ```shell
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: (same thing - this isn't a shell script).

File: content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md [hunk 3]
```diff
@@ -180,9 +197,28 @@ All code snippets in Windows sections are to be run in a PowerShell environment
...
+    After reboot, you can check via below command to check your docker service is ready.
 
-    ![alt_text](../windows-docker-error.png "windows docker error screen capture")    
+    ```PowerShell
+    docker version
+    ```
+
+    If you see error message like the following, you need to restart the docker service manually.
+
+    ```dummy
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: Replace `dummy` with the empty string here.

File: content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md [hunk 4]
```diff
@@ -180,9 +197,28 @@ All code snippets in Windows sections are to be run in a PowerShell environment
...
 
-    If after reboot you see the following error, you need to restart the docker service manually
+    After reboot, you can check via below command to check your docker service is ready.
 
-    ![alt_text](../windows-docker-error.png "windows docker error screen capture")    
+    ```PowerShell
+    docker version
+    ```
+
+    If you see error message like the following, you need to restart the docker service manually.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]:
```suggestion
    If you see error message like the following, you need to restart Docker manually.
```

Also: _start_, or _restart_? I'm not clear on what the reader should do.

Comment (yoonian) [reply content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: Yes, correct, it is `start`. I'll fix.

File: content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md (hunk 1 above)
Comment (yoonian) [reply content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: Okay, I'll apply.

File: content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md (hunk 2 above)
Comment (yoonian) [reply content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: Okay, I'll apply.

File: content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md (hunk 3 above)
Comment (yoonian) [reply content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: Okay, I'll apply.

**Comment by tengqm** [comment]
file name is "flannel-master-kubeclt-get-pods.png ", are you serious?

File: content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md [hunk 5]
```diff
@@ -180,9 +197,28 @@ All code snippets in Windows sections are to be run in a PowerShell environment
     [Environment]::SetEnvironmentVariable("HTTPS_PROXY", "http://proxy.example.com:443/", [EnvironmentVariableTarget]::Machine)
     ```
 
-    If after reboot you see the following error, you need to restart the docker service manually
+    After reboot, you can check via below command to check your docker service is ready.
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: Revise the sentence please, there are two "check" in it, which reads strange.

Comment (yoonian) [reply content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md]: Thanks. I'll apply.

**Pull request merged**